go/libraries/doltcore/merge/merge_prolly_rows.go

```diff
@@ -635,7 +635,7 @@ func (uv uniqValidator) validateDiff(ctx *sql.Context, diff tree.ThreeWayDiff) (
 
 // deleteArtifact deletes the unique constraint violation artifact for the row identified by |key| and returns a
 // boolean that indicates if an artifact was deleted, as well as an error that indicates if there were any
-// unexpected errors encountered.
+// unexpectededed errors encountered.
```

Comment:
'unexpectededed' is not a word, and it shows up lots of times in the PR. please fix.